subsys/infinispan/src/main/java/org/commonjava/indy/subsys/infinispan/CacheProducer.java

```diff
@@ -218,12 +243,31 @@ private EmbeddedCacheManager startCacheManager( EmbeddedCacheManager cacheMgr, S
     public synchronized <K, V> BasicCacheHandle<K, V> getBasicCache( String named )
     {
         BasicCacheHandle handle = caches.computeIfAbsent( named, ( k ) -> {
-            if ( remoteConfiguration.isEnabled() && remoteConfiguration.isRemoteCache( k ) )
+            if ( remoteConfiguration.isEnabled() )
             {
                 RemoteCache<K, V> cache = null;
                 try
                 {
-                    cache = remoteCacheManager.getCache( k );
+                    // For infinispan 9.x, it needs to load the specific cache configuration to create it
+                    // For infinispan 11.x, there is no need to load this configuration here, instead, declaring it
+                    // in hotrod-client.properties and get the cache by remoteCacheManager.getCache( "cacheName" )
+                    File confDir = indyConfiguration.getIndyConfDir();
+                    File cacheConf = new File( confDir, "cache-" + named + ".xml" );
```

Comment:
So, is this new per-cache XML configuration a feature of Infinispan 11, or it this just a new approach you're taking to allow us to configure different caches separately in configmaps?

Reply:
Yeah, it's new feature of Infinispan 11. 
https://infinispan.org/blog/2020/05/30/hotrod-percache-configuration/